include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //VkBool32                             shaderDenormPreserveFloat16;
+    //VkBool32                             shaderDenormPreserveFloat32;
+    //VkBool32                             shaderDenormPreserveFloat64;
+    //VkBool32                             shaderDenormFlushToZeroFloat16;
+    //VkBool32                             shaderDenormFlushToZeroFloat32;
+    //VkBool32                             shaderDenormFlushToZeroFloat64;
+    //VkBool32                             shaderRoundingModeRTEFloat16;
+    //VkBool32                             shaderRoundingModeRTEFloat32;
+    //VkBool32                             shaderRoundingModeRTEFloat64;
+    //VkBool32                             shaderRoundingModeRTZFloat16;
+    //VkBool32                             shaderRoundingModeRTZFloat32;
+    //VkBool32                             shaderRoundingModeRTZFloat64;
+    //uint32_t                             maxUpdateAfterBindDescriptorsInAllPools;
+    //VkBool32                             shaderUniformBufferArrayNonUniformIndexingNative;
+    //VkBool32                             shaderSampledImageArrayNonUniformIndexingNative;
+    //VkBool32                             shaderStorageBufferArrayNonUniformIndexingNative;
+    //VkBool32                             shaderStorageImageArrayNonUniformIndexingNative;
+    //VkBool32                             shaderInputAttachmentArrayNonUniformIndexingNative;
+    //VkBool32                             robustBufferAccessUpdateAfterBind;
+    //VkBool32                             quadDivergentImplicitLod;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindSamplers;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindUniformBuffers;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindStorageBuffers;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindSampledImages;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindStorageImages;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindInputAttachments;
+    //uint32_t                             maxPerStageUpdateAfterBindResources;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindSamplers;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindUniformBuffers;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindUniformBuffersDynamic;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindStorageBuffers;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindStorageBuffersDynamic;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindSampledImages;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindStorageImages;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindInputAttachments;
```

Comment:
in GL all values would be same as their equivalents in https://github.com/Devsh-Graphics-Programming/Nabla/pull/357/files#r887859625